testsuite/forge/src/interface/prometheus_metrics.rs

```diff
@@ -64,25 +66,38 @@ impl SystemMetrics {
     }
 }
 
-pub async fn fetch_error_metrics(
+pub async fn fetch_fullnode_failures(
     swarm: Arc<tokio::sync::RwLock<Box<dyn Swarm>>>,
 ) -> anyhow::Result<i64> {
-    let error_query = r#"aptos_error_log_count{role=~"validator"}"#;
+    let consensus_observer_failure_query = r#"consensus_observer_state_sync_fallback_counter"#;
```

Comment:
nit: Role might be good to add to the query to filter our unnecessary pods.
```suggestion
    let consensus_observer_failure_query = r#"consensus_observer_state_sync_fallback_counter{role=~"validator|fullnode"}"#;
```